.github/workflows/build-and-push.yml

```
@@ -26,6 +27,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
       - name: Log in to the Container registry
         uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

I'd like to use prod one if possible. (This is out-of-scope though)
```suggestion
        uses: docker/login-action@v2
```

And also for these.
- docker/metadata-action@v4
- docker/build-push-action@v4